docs/recipes.md

```diff
@@ -21,11 +21,11 @@ which live in the `src` folder:
 📁 repo
 └─╴📁 src
     └─╴📁 project
-        ├─╴📄 lorem
-        ├─╴📄 ipsum
-        ├─╴📄 dolor
-        ├─╴📄 sit
-        └─╴📄 amet
+        ├─╴📄 lorem.py
+        ├─╴📄 ipsum.py
+        ├─╴📄 dolor.py
+        ├─╴📄 sit.py
+        └─╴📄 amet.py
```

Comment:
I had intentionally left out the extensions since mkdocstrings is language agnostic. I'd prefer you send another PR with that change if you think it improves the docs :slightly_smiling_face:

Reply:
It didnt hit at the time that it is, will revert 😅
```suggestion
        ├─╴📄 lorem
        ├─╴📄 ipsum
        ├─╴📄 dolor
        ├─╴📄 sit
        └─╴📄 amet
```